src/components/Dashboard/AddressListSm.vue

```diff
@@ -68,9 +68,9 @@
         </v-expansion-panel-content>
       </v-expansion-panel>
 
-      <!--Records Office-->
+      <!-- Records Office -->
       <v-expansion-panel id="records-office-panel"
-        v-if="entityFilter(EntityTypes.BCOMP)"
+        v-if="isBComp()"
```

Comment:
I like it! The entityFilter might be a better choice if we have had to differentiate betweens 10's of entity types but this is simple and explicit 👍